### PR TITLE
Update ogc_client_support.rst

### LINKED ISSUE #697: [FEATURE] cache WMS capabililies
Original commit: https://github.com/qgis/QGIS/commit/9c340d7b4b8fb7bf03182155d2e883b084e31e8d by jef-n

Unfortunately this lazy coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by andremano**
fix #697

### CODE REVIEW

**Comment by yjacolin** [comment]
Do you know how if there is a way to clear cache or force not to use cache? It could be interesting to add this information. ping @jef-n

File: source/docs/user_manual/working_with_ogc/ogc_client_support.rst [hunk 1]
```diff
@@ -73,6 +73,9 @@ WMS layers can be added quite simply, as long as you know the URL to access
 the WMS server, you have a serviceable connection to that server, and the
 server understands HTTP as the data transport mechanism.
 
+Addicionally, QGIS will cache your WMS requests (i.e. images) for 24h.
```
Comment (tudorbarascu) [inline source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: @andremano Typo: You wrote Addicionally instead of Additionally

**Comment by tudorbarascu** [comment]
@andremano You're writing about QGIS Desktop WMS caching of tiles but the cachin in the #697 is on the server side which is caching the GetCapabilities request.
Can you make the typo adjustment and remove the reference to #697 is for caching the GetCapabilities request?

**Comment by yjacolin** [comment]
You probably asked to Jurgen, but I write here the answer to my question: Cache is cleared each time GetCapabilities is triggered. GC is triggered when the user open the 'open WMS layer' dialog and when selecting the layer.

If the user open again the QGIS project with the WMS layer, cache will be used if time < 24 hours.

**Comment by andremano** [comment]
Thanks Tudor, Jacolin

@tudor are u sure about that? The caching happens on the client side, no?

On Fri, May 27, 2016 at 4:29 PM, Jacolin <notifications@github.com> wrote:

> You probably asked to Jurgen, but I write here the answer to my question:
> Cache is cleared each time GetCapabilities is triggered. GC is triggered
> when the user open the 'open WMS layer' dialog and when selecting the layer.
>
> If the user open again the QGIS project with the WMS layer, cache will be
> used if time < 24 hours.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/qgis/QGIS-Documentation/pull/1050#issuecomment-222161157>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGW2kL2khlx4Ju9Ol9QUynmpsnFuU2tQks5qFv_WgaJpZM4IoYl6>
> .
>



-- 
..................................
André Mano
http://opussig.blogspot.com/

**Comment by yjacolin** [comment]
@andremano are you sure you mean @tudorbarascu and not me? All actions I talked about is on QGIS side (so client side)

**Comment by andremano** [comment]
Yes, I meant @tudorbarascu <https://github.com/tudorbarascu>, but it is
clear now. thanks

On Fri, May 27, 2016 at 5:51 PM, Jacolin <notifications@github.com> wrote:

> @andremano <https://github.com/andremano> are you sure you mean
> @tudorbarascu <https://github.com/tudorbarascu> and not me? All actions I
> talked about is on QGIS side (so client side)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/qgis/QGIS-Documentation/pull/1050#issuecomment-222183019>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGW2kMPLNgdu0CUdaPfH41kM2Z7x_KDVks5qFxLkgaJpZM4IoYl6>
> .
>



-- 
..................................
André Mano
http://opussig.blogspot.com/

**Comment by yjacolin** [comment]
LGTM as soon as tests passed

**Comment by tudorbarascu** [comment]
@andremano Thanks! You were right!

**Pull request merged**